kong/router/atc.lua

```diff
@@ -382,12 +383,17 @@ function _M:matching(params)
 
   local request_prefix = matched_route.strip_path and matched_path or nil
 
+  local uri_captures = nil
+  if matched_path and captures and tb_nkeys(captures) > 1 then
```

Comment:
As we talked before, I suggest to use `>= 2` and add some comments to explain it.

Reply:
Done.